app/helpers/attendances_helper.rb

```diff
@@ -0,0 +1,2 @@
+module AttendancesHelper
+end
```

Comment:
If you don't use it, you can remove it.

Reply:
Yes, we thought about removing it, but we keep it in case we need it in the future. If we don't use it, we remove it in the next features.